src/convnets/resnets/core.jl

```diff
@@ -291,35 +293,46 @@ function resnet(img_dims, stem, get_layers, block_repeats::AbstractVector{<:Inte
     return Chain(backbone, classifier)
 end
 
-function resnet(block_type::Symbol, block_repeats::AbstractVector{<:Integer};
-                downsample_opt::NTuple{2, Any} = (downsample_conv, downsample_identity),
+function resnet(block_type::Symbol, block_repeats::AbstractVector{<:Integer},
```

Comment:
Here, it doesn't matter because the builder has to be hardcoded in the body of `resnet` - making it a `Symbol` I thought was a clean way of indicating that not all values for `block_type` would work

Reply:
Yeah but an error would be thrown immediately anyways? I think of a `Symbol` like a string...meaning I need to look up the accepted values (i.e. how should they be spelled, etc.). For the function directly, I feel it is more intuitive what the accepted values are. I see your point though; it's up to you.